iot-hub/Samples/service/EdgeDeploymentSample/EdgeDeploymentSample.cs

```diff
@@ -80,29 +80,29 @@ public async Task RunSampleAsync()
         private static IEnumerable<Device> GenerateEdgeDevices(string deviceIdPrefix, int numToAdd)
         {
             IList<Device> edgeDevices = new List<Device>();
-            
+
             for (var i = 0; i < numToAdd; i++)
             {
-                var deviceName = $"{deviceIdPrefix}_{i:D8}-{Guid.NewGuid().ToString()}";
+                string deviceName = $"{deviceIdPrefix}_{i:D8}-{Guid.NewGuid().ToString()}";
                 var device = new Device(deviceName)
                 {
                     Capabilities = new DeviceCapabilities
                     {
-                        IotEdge = true
+                        IotEdge = true,
```

Comment:
I have seen you usually adding "," for initializers like this one. Just wondering why?

Reply:
It is a good practice to follow, and C# allows trailing commas.

1. When you need to add another property, the last property listed doesn't need to change, because it already has a comma.
1. If you want to remark out or reorder any of the properties, none of the other lines need to change.